modules/gateway/peers.go

```diff
@@ -17,6 +18,15 @@ const (
 	fullyConnectedThreshold = 128
 	// the gateway will ask for more addresses below this threshold
 	minNodeListLen = 100
+	// minAcceptableVersion is the version below which the gateway will refuse to
+	// connect to peers and reject connection attempts.
+	//
+	// Reject peers < v0.4.0 as the previous version is v0.3.3 which is
+	// pre-hardfork.
+	//
+	// NOTE: this version must be bumped whenever the gateway or consensus
+	// breaks compatibility.
```

Comment:
NOTE is not necessary, also I would prefer if there were newlines between the consts.